Resources/config/ckeditor.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="cmf_content.form.extension.ckeditor"
+            class="Symfony\Cmf\Bundle\ContentBundle\Form\Extension\CKEditorExtension">
```

Comment:
In the Configuration, we only allow to pass the config_name as option (https://github.com/symfony-cmf/content-bundle/blob/master/DependencyInjection/CmfContentExtension.php#L63-L65). As it uses a parameter, anyone could override it. I can reintroduce the same code and use this parameter as extension argument.